spyderlib/spyder.py

```diff
@@ -1144,7 +1146,11 @@ def add_xydoc(text, pathlist):
                                          self.close_dockwidget_action,
                                          self.maximize_action,
                                          None))
+            self.toggle_toolbars_action = create_action(self,
+                                    _("Toggle toolbar visibility"),
+                                    toggled=self.toggle_toolbars)

```

Comment:
Check my previous comment, I would rather have something like `show_hide_toolbars` and change the entry in the menu from `Show toolbars` -> `Hide toolbars` depending on the state.